examples/speaker_tasks/diarization/conf/inference/diar_infer_meeting.yaml

```diff
@@ -6,7 +6,7 @@
 # {"audio_filepath": "/path/to/audio_file", "offset": 0, "duration": null, "label": "infer", "text": "-", "num_speakers": null, "rttm_filepath": "/path/to/rttm/file", "uem_filepath": "/path/to/uem/file"}
 name: &name "ClusterDiarizer"
 
-num_workers: 4
+num_workers: 1
```

Comment:
Could you add a comment say they can use larger values ?

Reply:
But that doesn't work now. So its better to add when we fix multiprocessing issue